styles.css

```diff
@@ -85,6 +85,11 @@ body::after {
     margin-top: 20px;
     animation: fadeInGif ease-in 5s;
     width: 300px;
+    height: 249px;
+    background: url("https://d.wattpad.com/story_parts/837710361/images/15f26e80b98996db696933165236.gif");
+    background-repeat: no-repeat;
+    background-position: center;
+    display: none;
```

Comment:
Ik ben benieuwd waarom je hier voor 'display:none' kiest, terwijl er op regel 83 al 'display:flex' staat. Ik neem aan dat regel 83 overbodig is?